__tests__/app.test.js

```diff
@@ -114,12 +147,10 @@ describe('API Routes', () => {
   // If a GET request is made to /api/cats/:id, does:
   // 1) the server respond with status of 200
   // 2) the body match the expected API data for the cat with that id?
-  test('GET /api/albums/:id', async () => {
+  test.skip('GET /api/albums/:id', async () => {
```

Comment:
On line 153 you still have expectedAlbums and it longer exists now that you've commented it out up above. I think this is causing Github CI to fail your tests.